modules/machine-health-checks-pausing.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+
+// * updating/updating-cluster-cli.adoc
+
+[id="machine-health-checks-pausing_{context}"]
+= Pausing a MachineHealthCheck resource
+
+During the upgrade process, nodes in the cluster might become temporarily unavailable. The `MachineHealthCheck` might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, pause all the `MachineHealthCheck` resources before updating the cluster.
+
+.Prerequisites
+
+* Install the `oc` command line interface.
+
+.Procedure
+
+* To pause the machine health checks, add `cluster.x-k8s.io/paused=""` annotation to the `MachineHealthCheck` resource. To do so, run the following command:
++
+[source,terminal]
+----
+$ oc -n <namespace> annotate mhc <mhc-name> cluster.x-k8s.io/paused=""
+----
+
+The annotated machine health check resource resembles the following YAML file:
+
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineHealthCheck
+metadata:
+  name: example
+  namespace: openshift-machine-api
+  annotations:
+    cluster.x-k8s.io/paused: ""
+spec:
+  selector:
+    matchLabels:
+      role: worker
+  unhealthyConditions:
+  - type:    "Ready"
+    status:  "Unknown"
+    timeout: "300s"
+  - type:    "Ready"
+    status:  "False"
+    timeout: "300s"
+  maxUnhealthy: "40%"
+status:
+  currentHealthy: 5
+  expectedMachines: 5
+----
+
+[IMPORTANT]
+====
+Resume the machine health checks after updating the cluster. To resume the check, remove the pause annotation from the `MachineHealthCheck` resource as follows:
```

Comment:
This is purely a suggestion because I think it's clearer to be explicit about running commands:

```suggestion
Resume the machine health checks after updating the cluster. To resume the check, remove the pause annotation from the `MachineHealthCheck` resource by running the following command:
```